src/debug.cpp

```diff
@@ -118,6 +118,21 @@ void printConnectionMap(const ConnectionMap &map)
     }
 }
 
+void printComponentMap(const ComponentMap &map)
```

Comment:
what am I missing? Isn't this adding a method that is not used anywhere?

Reply:
You are probably missing that it is a debugging function for debugging only.  We keep some debugging code around in case it is useful, but we don't allow it to be included. We have a test that determines if the debug header file is included to guard against accidental inclusion.

Reply:
(just weird to see a new unused method added in a PR that addresses coverage...)

Reply:
I had to use it to help me figure out if I could trigger the case I was after.